eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -69,6 +69,8 @@ parameters:
     ezsettings.default.content.default_ttl: 60          # Default TTL cache value for content
     ezsettings.default.content.tree_root.location_id: 2 # Root locationId for routing and link generation. Useful for multisite apps with one repository.
     ezsettings.default.content.tree_root.excluded_uri_prefixes: [] # URI prefixes that are allowed to be outside the content tree
+    ezsettings.default.content.field_groups.list: ['content', 'metadata']
```

Comment:
need doc somewhere that this settings needs to be global to repo somewhere, but can also be mentioned here.